src/Domain/Structure/Neighbors.hpp

```diff
@@ -98,9 +118,19 @@ class Neighbors {
     return ids_.end();
   }
 
+  std::ostream& print(std::ostream& os) const;
+
  private:
-  std::unordered_set<IdType> ids_;
-  OrientationMap<VolumeDim> orientation_;
+  void maximum_neighbor_check();
+  template <size_t LocalVolumeDim, typename LocalIdType>
+  // NOLINTNEXTLINE(readability-redundant-declaration)
+  friend bool operator==(const Neighbors<LocalVolumeDim, LocalIdType>& lhs,
+                         const Neighbors<LocalVolumeDim, LocalIdType>& rhs);
+
+  std::unordered_set<IdType> ids_{};
+  domain::InterfaceDataPolicy interface_data_policy_{
```

Comment:
I'm a bit hesitant about storing this enum here because it couples details of our DG scheme into the domain structure. The domain structure should not need to know about things like projecting and interpolating. What do you think about storing this interface data policy in an option tag instead, which goes into the discretization group? We probably want to do the same policy on all cube/sphere boundaries, so there's no need to store it separately at each interface. Then I think on the domain side of things we just need to store a `domain::BlockGeometry` enum with cases `Cube` and `SphericalShell` to have all the information we need.

Reply:
- It has nothing to do with DG specifically.  It just enables a switch based on how the coordinates on the two sides of the interface are related.
- The information needs to be in the Element just like the orientation as that is what is available at the decision points.
- No there is not a global value for the policy.  It is boundary dependent.  Especially as we are adding cylinders as well.
- There is no need for a BlockGeometry enum as the Basis and Quadrature already encode the information.

Reply:
Just to quickly clarify, I mean there's a global value for how cube/sphere interfaces are handled, and another global value for how cylinder interfaces are handled. We don't want to make this decision differently at each interface.